docs/toc.html

```diff
@@ -22,61 +22,62 @@
     <ul class="toc">
         <li><a href="#gettingStarted">1. Getting Started</a>
             <ul>
-                <li><a href="#introduction">1.1 Introduction</a>
-                <li><a href="#uses">1.2 Use Cases</a>
-                <li><a href="#quickstart">1.3 Quick Start</a>
-                <li><a href="#ecosystem">1.4 Ecosystem</a>
-                <li><a href="#upgrade">1.5 Upgrading</a>
-                <li><a href="#docker">1.6 Docker</a>
+                <li><a href="#introduction">1.1 Introduction</a></li>
```

Comment:
As per the HTML spec, the closing `li` tag is optional: https://www.w3.org/TR/2012/WD-html-markup-20121025/li.html#li-tags

So let's undo these changes.